flake.nix

```diff
@@ -0,0 +1,47 @@
+# SPDX-FileCopyrightText: 2021 Serokell <https://serokell.io/>
```

Comment:
Taken from https://serokell.io/blog/practical-nix-flakes#packaging-existing-applications - works pretty well for this but not that familiar with nix